solidity/contracts/libs/Call.sol

```diff
@@ -4,51 +4,109 @@ pragma solidity ^0.8.13;
 import {Address} from "@openzeppelin/contracts/utils/Address.sol";
 
 library CallLib {
+    struct StaticCall {
+        address to;
```

Comment:
Can we realistically support non-evm chains if this isn't bytes32? Flagging b/c we switched to bytes32 elsewhere, but is that useful if our messages don't support bytes32 here?

Reply:
Oh I guess you could control ICAs on EVM chains via accounts on non-EVM chains, but not vice versa, is that right?

Reply:
I guess Call.sol is specific to EVM land
we could make InterchainCallMessage encode addresses as bytes32